common/common-rest/src/main/java/io/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -126,12 +129,15 @@ protected void scheduleInvocation() {
     });
   }
 
-  protected void runOnExecutor(QueueMetrics metricsData) {
+  protected void runOnExecutor(InvocationStartedEvent startedEvent) {
     Object[] args = RestCodec.restToArgs(requestEx, restOperationMeta);
     createInvocation(args);
 
-    this.invocation.setMetricsData(metricsData);
-    updateMetrics();
+    long startProcessingTime = System.nanoTime();
```

Comment:
Suggestion: "System.nanoTime()" we can use directly at below line.
this.invocation.setStartProcessingTime(System.nanoTime());

Reply:
we need use same value below ,so may save it in a variable first better